### ISSUE TITLE
This is awesome

### ISSUE THREAD

**Issue by OptiSean** [PeterMalkin/goxtoolbots#2]
This is cool Peter. Are you still interested in crypto projects? :)

**Comment by PeterMalkin** [comment]
Yes, I am. As a matter of fact, I am currently working on a faster miner
for BIS crypto currency.

But as far as trading - I was working on porting the bot to work on gdax,
but it's not ready yet.

**Comment by OptiSean** [comment]
Hey cool coin, I don’t fully grasp BIS after jus researching a bit but I would imagine the goal is maybe cheaper more sustainable POW model? :)

**Comment by OptiSean** [comment]
Is there any chance you’d port it to work on bitfinex on multiple coins? I see you are busy but I thought it can’t hurt to ask, I’m working on a project incorporating some automation of trading based on a pretty cool formula would love to share with you some more some time maybe if that’s up your alley of interest.
